microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,160 @@ These Release Notes provide information about new features and bug fixes that ar
 the templates to copy for incoming sections are in ./includes/templates.md
 
 this webpage covers the most recent 6 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers:
+Dec 2025
 Nov 2025
 Oct 2025
 Sep 2025
 Aug 2025
 Jul 2025
-Jun 2025
 
 moved to archive: 
-May 2025
+Jun 2025
 -->
 
 
+<!-- Dec 2025 Release SDK -->
+<!-- ====================================================================== -->
+## 1.0.3650.58
+
+Release Date: December 8, 2025
+
+[NuGet package for WebView2 SDK 1.0.3650.58](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3650.58)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 143.0.3650.58 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions to Phase 3: Stable in Release
+
+No additional APIs have been promoted from Phase 2: Stable in Prerelease, to Phase 3: Stable in Release, in this Release SDK.
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+###### Runtime-only
+
+* Fixed local network access triggering a permission alert pop-up window.
+
+<!-- end of Dec 2025 Release SDK -->
+
+
+<!-- Dec 2025 Prerelease SDK -->
+<!-- ====================================================================== -->
+## 1.0.3712-prerelease
+
+Release Date: December 8, 2025
+
+[NuGet package for WebView2 SDK 1.0.3712-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3712-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 144.0.3712.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+The following APIs are in Phase 1: Experimental in Prerelease, and have been added in this Prerelease SDK.
+
+
+<!-- ---------- -->
+###### PortRange
+
+todo: Description.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2AllowedPortRangeScope Class](/dotnet/api/microsoft.web.webview2.core.corewebview2allowedportrangescope?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+   * `CoreWebView2AllowedPortRangeScope.Default`
+   * `CoreWebView2AllowedPortRangeScope.WebRtc`
+
+* [CoreWebView2TransportProtocolKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2transportprotocolkind?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+*  `CoreWebView2TransportProtocolKind.Udp`
+
+* [CoreWebView2EnvironmentOptions.GetEffectiveAllowedPortRange Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.geteffectiveallowedportrange?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+* [CoreWebView2EnvironmentOptions.SetAllowedPortRange Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.setallowedportrange?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+N/A<!-- todo: are these eventually planned? -->
```

Comment:
do we plan to add APIs for RT like for .NET, eg GetEffectiveAllowedPortRange, eventually?
@Ju-MsGit